dotenv/parse_test.ts

```diff
@@ -3,11 +3,14 @@
 import { assertEquals } from "@std/assert";
 import { parse } from "./parse.ts";
 import * as path from "@std/path";
+import { assertSpyCall, spy } from "@std/testing/mock";
 
 const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
 const testdataDir = path.resolve(moduleDir, "testdata");
 
 Deno.test("parse()", () => {
+  using consoleWarnSpy = spy(console, "warn");
```

Comment:
Wow - great idea! I wonder where else in the codebase we can spy on `console` instead of spawning new processes and reading the output.